app/src/ui/lib/list/list.tsx

```diff
@@ -1144,10 +1155,13 @@ export class List extends React.Component<IListProps, IListState> {
     }
   }
 
-  private onRowMouseDown = (row: number, event: React.MouseEvent<any>) => {
-    if (this.canSelectRow(row)) {
+  private onRowMouseDown = (
+    indexPath: RowIndexPath,
+    event: React.MouseEvent<any>
+  ) => {
+    if (this.canSelectRow(indexPath.row)) {
```

Comment:
Just a thought I had from looking at this. If we did a `const {row} = rowIndexPath` at the top, wouldn't have to replacements in this method.